### DISCUSSION TITLE
My TFT35 V2 White screen

### DISCUSSION THREAD

**Discussion by manuelchaves** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1777] [General]
My display only shows a white screen, even when only 5V is used.
Any help? Tks
![WhatsApp Image 2021-03-26 at 18 56 56](https://user-images.githubusercontent.com/8344682/112696815-48d9b380-8e65-11eb-8dc5-abed4f868a47.jpeg)

**Comment by oldman4U** [comment]
Reset with reset.txt file?

**Reply by manuelchaves** [reply]
Hi, I don't know how to do ...
Tks

**Reply by oldman4U** [reply]
ReadMe on main page of this repository.

**Comment by oldman4U** [comment]
Has it been like this from the beginning? Or after firmware update?

**Comment by manuelchaves** [comment]
From the beginning...

**Comment by oldman4U** [comment]
Most likely a broken hardware. If the reset and firmware update does not work I would contact the dealer you bought it from.

Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by manuelchaves** [comment]
tks
